discord_thread_summarizer_using_openai/main.bal

```diff
@@ -0,0 +1,76 @@
+import ballerina/io;
+import ballerina/time;
+import ballerina/log;
+
+const TYPE_FORUM_CHANEL = 15;
+
+configurable int forumChannelId = ?;
+
+function readThreads(ActiveThreads activeThreads) returns error? {
+
+    foreach ChannelThread thread in activeThreads.threads {
+
+        // check if the thread has any recent messages
+        if !check checkRecentMessages(thread) {
+            continue;
+        }
+
+        string prompt = check constructPrompt(thread);
+        io:println(prompt);
+
+        string|error generateChatCompletionResult = generateChatCompletion(prompt, model = "gpt-3.5-turbo");
```

Comment:
Can remote the second param here.

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/ca9ecd0b4475145df3519080000a28a26d049d7d